<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvelle Partie</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelJoueur1">
         <property name="text">
          <string>Nom du joueur 1 :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditJoueur1"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelJoueur2">
         <property name="text">
          <string>Nom du joueur 2 :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditJoueur2"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelSets">
         <property name="text">
          <string>Nombre de sets gagants :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton1set">
         <property name="text">
          <string>1 set gagnant</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">groupSet</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton2set">
         <property name="text">
          <string>2 sets gagnants</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">groupSet</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="radioButton3set">
         <property name="text">
          <string>3 sets gagnants</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">groupSet</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelPoints">
         <property name="text">
          <string>Nombre de points par set :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton11points">
           <property name="text">
            <string>11 points</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">groupPoint</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton21points">
           <property name="text">
            <string>21 points</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">groupPoint</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonAutre">
           <property name="text">
            <string>autre :</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">groupPoint</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditAutre">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="boutonOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonAnnuler">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="groupSet"/>
  <buttongroup name="groupPoint"/>
 </buttongroups>
</ui>
